hexa/workspaces/schema/mutations.py

```
@@ -150,13 +150,19 @@ def resolve_invite_workspace_member(_, info, **kwargs):
                 "workspace_membership": workspace_membership,
             }
         except User.DoesNotExist:
-            # If the user does not exist, we create an invitation to create an account and join the workspace
-            invitation = WorkspaceInvitation.objects.create_if_has_perm(
-                principal=request.user,
-                workspace=workspace,
-                email=input["userEmail"],
-                role=input["role"],
-            )
+            # If the user does not exist we first
```

Better to let the code crash thanks to a unique constraint